### ISSUE TITLE
[Bug]: Broken example links in Python Tutorial

### ISSUE THREAD

**Issue by yamasite** [milvus-io/milvus#11613]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Broken example links in https://milvus.io/api-reference/pymilvus/v2.0.0rc8/tutorial.html

- https://github.com/milvus-io/pymilvus/blob/main/examples/example.py
- https://github.com/milvus-io/pymilvus/blob/main/examples/collection.py
- https://github.com/milvus-io/pymilvus/blob/main/examples/example_index.py

### Expected Behavior

"main" should be "master" to make the links right

### Anything else?

An automatic link checker integrated into the ci-cd process is recommended. Perhaps use this? https://github.com/yamasite/online-doc-link-checker

**Comment by yanliang567** [comment]
@shanghaikid  could you please help to fix it

/assign @shanghaikid 
/unassign

**Comment by XuanYang-cn** [comment]
fix broken link in doc milvus-io/pymilvus#812

**Comment by xiaofan-luan** [comment]
closed for now
